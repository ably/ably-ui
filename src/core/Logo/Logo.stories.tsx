import Logo from "../Logo";
import logoUrl from "../images/ably-logo.png";

export default {
  title: "Components/Logo",
  component: Logo,
  args: {
    logoUrl,
  },
};

export const Default = {};
