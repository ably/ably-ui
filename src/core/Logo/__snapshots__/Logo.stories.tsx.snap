// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Logo Default smoke-test 1`] = `
<div class="grid grid-cols-1 sm:grid-cols-2 gap-4">
  <div class="text-center ui-text-code2 order-1">
    light
  </div>
  <div class="text-center ui-text-code2 order-3 sm:order-2">
    dark
  </div>
  <a href="/"
     class="flex items-center gap-2 justify-center text-center p-2 bg-neutral-000 rounded order-2 sm:order-3"
  >
    <img src="data:image/svg+xml,%3csvg%20width='204'%20height='64'%20viewBox='0%200%20204%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M125.844%2049.9572V8.17627H133.387V23.3024C135.942%2020.87%20139.233%2019.5209%20142.708%2019.5209C150.864%2019.5209%20158.1%2025.7349%20158.1%2035.0967C158.1%2044.4586%20150.864%2050.693%20142.708%2050.693C139.049%2050.693%20135.594%2049.2417%20133.019%2046.6049V49.9572H125.844ZM150.557%2035.0967C150.557%2029.8639%20146.837%2026.205%20141.972%2026.205C137.23%2026.205%20133.571%2029.6799%20133.387%2034.7288V35.0967C133.387%2040.3296%20137.107%2043.9885%20141.972%2043.9885C146.837%2043.9885%20150.557%2040.3296%20150.557%2035.0967ZM161.595%2049.9572V8.17627H169.138V49.9572H161.595ZM179.685%2060.7908L184.08%2050.3047L172.142%2020.2568H180.298L187.902%2041.2494L195.629%2020.2568H203.907L187.718%2060.8112H179.685V60.7908ZM113.866%2020.2568V24.0383C111.249%2021.1766%20107.59%2019.5413%20103.829%2019.5413C95.6735%2019.5413%2088.4374%2025.7553%2088.4374%2035.1172C88.4374%2044.4995%2095.6735%2050.693%20103.829%2050.693C107.734%2050.693%20111.433%2048.9965%20114.111%2045.9508V49.9776H120.693V20.2568H113.866ZM113.13%2035.0967C113.13%2040.2683%20109.41%2043.9885%20104.545%2043.9885C99.6799%2043.9885%2095.9596%2040.2683%2095.9596%2035.0967C95.9596%2029.9252%2099.6799%2026.205%20104.545%2026.205C109.287%2026.205%20112.946%2029.7413%20113.13%2034.7288V35.0967Z'%20fill='%2303020D'/%3e%3cpath%20d='M38.5717%200L6.29575%2059.0738L0%2054.6586L29.8639%200H38.5717ZM39.0214%200L71.2973%2059.0738L77.5931%2054.6586L47.7292%200H39.0214Z'%20fill='url(%23paint0_linear_991_548)'/%3e%3cpath%20d='M70.8476%2059.4213L38.7965%2034.3201L6.74542%2059.4213L13.2865%2064L38.7965%2044.0294L64.3066%2064L70.8476%2059.4213Z'%20fill='url(%23paint1_linear_991_548)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_548'%20x1='10.9472'%20y1='74.8439'%20x2='64.9206'%20y2='14.9005'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_548'%20x1='21.4168'%20y1='78.7187'%20x2='53.3166'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
  <a href="/"
     class="flex items-center gap-2 justify-center ui-theme-dark text-center p-2 bg-neutral-1300 rounded order-4"
  >
    <img src="data:image/svg+xml,%3csvg%20width='205'%20height='64'%20viewBox='0%200%20205%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M126.751%2049.9572V8.17627H134.294V23.3024C136.849%2020.87%20140.14%2019.5209%20143.615%2019.5209C151.771%2019.5209%20159.007%2025.7349%20159.007%2035.0967C159.007%2044.4586%20151.771%2050.693%20143.615%2050.693C139.956%2050.693%20136.501%2049.2417%20133.926%2046.6049V49.9572H126.751ZM151.464%2035.0967C151.464%2029.8639%20147.744%2026.205%20142.879%2026.205C138.137%2026.205%20134.478%2029.6799%20134.294%2034.7288V35.0967C134.294%2040.3296%20138.014%2043.9885%20142.879%2043.9885C147.744%2043.9885%20151.464%2040.3296%20151.464%2035.0967ZM162.502%2049.9572V8.17627H170.045V49.9572H162.502ZM180.592%2060.7908L184.987%2050.3047L173.05%2020.2568H181.205L188.809%2041.2494L196.536%2020.2568H204.814L188.625%2060.8112H180.592V60.7908ZM114.773%2020.2568V24.0383C112.157%2021.1766%20108.498%2019.5413%20104.737%2019.5413C96.5807%2019.5413%2089.3447%2025.7553%2089.3447%2035.1172C89.3447%2044.4995%2096.5807%2050.693%20104.737%2050.693C108.641%2050.693%20112.341%2048.9965%20115.018%2045.9508V49.9776H121.6V20.2568H114.773ZM114.037%2035.0967C114.037%2040.2683%20110.317%2043.9885%20105.452%2043.9885C100.587%2043.9885%2096.8669%2040.2683%2096.8669%2035.0967C96.8669%2029.9252%20100.587%2026.205%20105.452%2026.205C110.194%2026.205%20113.853%2029.7413%20114.037%2034.7288V35.0967Z'%20fill='white'/%3e%3cpath%20d='M39.4789%200L7.20298%2059.0738L0.907227%2054.6586L30.7712%200H39.4789ZM39.9286%200L72.2046%2059.0738L78.5003%2054.6586L48.6364%200H39.9286Z'%20fill='url(%23paint0_linear_991_544)'/%3e%3cpath%20d='M71.7548%2059.4213L39.7037%2034.3201L7.65265%2059.4213L14.1937%2064L39.7037%2044.0294L65.2138%2064L71.7548%2059.4213Z'%20fill='url(%23paint1_linear_991_544)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_544'%20x1='11.8545'%20y1='74.8439'%20x2='65.8278'%20y2='14.9005'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_544'%20x1='22.324'%20y1='78.7187'%20x2='54.2238'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
</div>
`;

exports[`Components/Logo Mono smoke-test 1`] = `
<div class="grid grid-cols-1 sm:grid-cols-2 gap-4">
  <div class="text-center ui-text-code2 order-1">
    light
  </div>
  <div class="text-center ui-text-code2 order-3 sm:order-2">
    dark
  </div>
  <a href="/"
     class="flex items-center gap-2 justify-center text-center p-2 bg-neutral-000 rounded order-2 sm:order-3"
  >
    <img src="data:image/svg+xml,%3csvg%20width='204'%20height='64'%20viewBox='0%200%20204%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M125.844%2049.9572V8.17627H133.387V23.3024C135.942%2020.87%20139.233%2019.5209%20142.708%2019.5209C150.864%2019.5209%20158.1%2025.7349%20158.1%2035.0967C158.1%2044.4586%20150.864%2050.693%20142.708%2050.693C139.049%2050.693%20135.594%2049.2417%20133.019%2046.6049V49.9572H125.844ZM150.557%2035.0967C150.557%2029.8639%20146.837%2026.205%20141.972%2026.205C137.23%2026.205%20133.571%2029.6799%20133.387%2034.7288V35.0967C133.387%2040.3296%20137.107%2043.9885%20141.972%2043.9885C146.837%2043.9885%20150.557%2040.3296%20150.557%2035.0967ZM161.595%2049.9572V8.17627H169.138V49.9572H161.595ZM179.685%2060.7908L184.08%2050.3047L172.142%2020.2568H180.298L187.902%2041.2494L195.629%2020.2568H203.907L187.718%2060.8112H179.685V60.7908ZM113.866%2020.2568V24.0383C111.249%2021.1766%20107.59%2019.5413%20103.829%2019.5413C95.6735%2019.5413%2088.4374%2025.7553%2088.4374%2035.1172C88.4374%2044.4995%2095.6735%2050.693%20103.829%2050.693C107.734%2050.693%20111.433%2048.9965%20114.111%2045.9508V49.9776H120.693V20.2568H113.866ZM113.13%2035.0967C113.13%2040.2683%20109.41%2043.9885%20104.545%2043.9885C99.6799%2043.9885%2095.9596%2040.2683%2095.9596%2035.0967C95.9596%2029.9252%2099.6799%2026.205%20104.545%2026.205C109.287%2026.205%20112.946%2029.7413%20113.13%2034.7288V35.0967Z'%20fill='%2303020D'/%3e%3cpath%20d='M38.5717%200L6.29575%2059.0738L0%2054.6586L29.8639%200H38.5717ZM39.0214%200L71.2973%2059.0738L77.5931%2054.6586L47.7292%200H39.0214Z'%20fill='url(%23paint0_linear_991_540)'/%3e%3cpath%20d='M70.8476%2059.4213L38.7965%2034.3201L6.74542%2059.4213L13.2865%2064L38.7965%2044.0294L64.3066%2064L70.8476%2059.4213Z'%20fill='url(%23paint1_linear_991_540)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_540'%20x1='17.7591'%20y1='88.3691'%20x2='76.9824'%20y2='48.3678'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23292831'/%3e%3cstop%20offset='0.6875'%20stop-color='%231D1C25'/%3e%3cstop%20offset='1'%20stop-color='%2303020D'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_540'%20x1='21.4168'%20y1='78.7187'%20x2='53.3166'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23292831'/%3e%3cstop%20offset='0.6875'%20stop-color='%231D1C25'/%3e%3cstop%20offset='1'%20stop-color='%2303020D'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
  <a href="/"
     class="flex items-center gap-2 justify-center ui-theme-dark text-center p-2 bg-neutral-1300 rounded order-4"
  >
    <img src="data:image/svg+xml,%3csvg%20width='205'%20height='64'%20viewBox='0%200%20205%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M126.751%2049.9572V8.17627H134.294V23.3024C136.849%2020.87%20140.14%2019.5209%20143.615%2019.5209C151.771%2019.5209%20159.007%2025.7349%20159.007%2035.0967C159.007%2044.4586%20151.771%2050.693%20143.615%2050.693C139.956%2050.693%20136.501%2049.2417%20133.926%2046.6049V49.9572H126.751ZM151.464%2035.0967C151.464%2029.8639%20147.744%2026.205%20142.879%2026.205C138.137%2026.205%20134.478%2029.6799%20134.294%2034.7288V35.0967C134.294%2040.3296%20138.014%2043.9885%20142.879%2043.9885C147.744%2043.9885%20151.464%2040.3296%20151.464%2035.0967ZM162.502%2049.9572V8.17627H170.045V49.9572H162.502ZM180.592%2060.7908L184.987%2050.3047L173.05%2020.2568H181.205L188.809%2041.2494L196.536%2020.2568H204.814L188.625%2060.8112H180.592V60.7908ZM114.773%2020.2568V24.0383C112.157%2021.1766%20108.498%2019.5413%20104.737%2019.5413C96.5807%2019.5413%2089.3447%2025.7553%2089.3447%2035.1172C89.3447%2044.4995%2096.5807%2050.693%20104.737%2050.693C108.641%2050.693%20112.341%2048.9965%20115.018%2045.9508V49.9776H121.6V20.2568H114.773ZM114.037%2035.0967C114.037%2040.2683%20110.317%2043.9885%20105.452%2043.9885C100.587%2043.9885%2096.8669%2040.2683%2096.8669%2035.0967C96.8669%2029.9252%20100.587%2026.205%20105.452%2026.205C110.194%2026.205%20113.853%2029.7413%20114.037%2034.7288V35.0967Z'%20fill='white'/%3e%3cpath%20d='M39.4789%200L7.20298%2059.0738L0.907227%2054.6586L30.7712%200H39.4789ZM39.9286%200L72.2046%2059.0738L78.5003%2054.6586L48.6364%200H39.9286Z'%20fill='url(%23paint0_linear_991_536)'/%3e%3cpath%20d='M71.7548%2059.4213L39.7037%2034.3201L7.65265%2059.4213L14.1937%2064L39.7037%2044.0294L65.2138%2064L71.7548%2059.4213Z'%20fill='url(%23paint1_linear_991_536)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_536'%20x1='18.6663'%20y1='88.3691'%20x2='77.8896'%20y2='48.3678'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23E5E5E5'/%3e%3cstop%20offset='0.6875'%20stop-color='%23E5E5E5'/%3e%3cstop%20offset='1'%20stop-color='white'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_536'%20x1='22.324'%20y1='78.7187'%20x2='54.2238'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23E5E5E5'/%3e%3cstop%20offset='0.6875'%20stop-color='%23E5E5E5'/%3e%3cstop%20offset='1'%20stop-color='white'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
</div>
`;

exports[`Components/Logo Stacked smoke-test 1`] = `
<div class="grid grid-cols-1 sm:grid-cols-2 gap-4">
  <div class="text-center ui-text-code2 order-1">
    light
  </div>
  <div class="text-center ui-text-code2 order-3 sm:order-2">
    dark
  </div>
  <a href="/"
     class="flex items-center gap-2 justify-center text-center p-2 bg-neutral-000 rounded order-2 sm:order-3"
  >
    <img src="data:image/svg+xml,%3csvg%20width='119'%20height='128'%20viewBox='0%200%20119%20128'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M38.5427%20116.818V73.7886H46.3261V89.3552C48.9432%2086.8401%2052.342%2085.4805%2055.9108%2085.4805C64.3059%2085.4805%2071.7833%2091.8704%2071.7833%20101.523C71.7833%20111.176%2064.3059%20117.566%2055.9108%20117.566C52.1381%20117.566%2048.5693%20116.07%2045.9522%20113.351V116.784H38.5427V116.818ZM64%20101.523C64%2096.1189%2060.1593%2092.3802%2055.163%2092.3802C50.2687%2092.3802%2046.496%2095.9489%2046.3261%20101.149V101.523C46.3261%20106.927%2050.1667%20110.666%2055.163%20110.666C60.1593%20110.666%2064%20106.927%2064%20101.523ZM75.3521%20116.818V73.7886H83.1354V116.818H75.3521ZM94.0117%20128L98.5321%20117.192L86.2283%2086.2283H94.6234L102.475%20107.845L110.428%2086.2283H118.959L102.271%20128H94.0117ZM26.205%2086.2283V90.103C23.5199%2087.146%2019.7472%2085.4805%2015.8725%2085.4805C7.44344%2085.4805%200%2091.8704%200%20101.523C0%20111.176%207.44344%20117.566%2015.8725%20117.566C19.8832%20117.566%2023.7238%20115.798%2026.4769%20112.671V116.818H33.2406V86.2283H26.205ZM25.4572%20101.523C25.4572%20106.859%2021.6166%20110.666%2016.6203%20110.666C11.624%20110.666%207.78332%20106.825%207.78332%20101.523C7.78332%2096.2209%2011.624%2092.3802%2016.6203%2092.3802C21.5146%2092.3802%2025.2873%2096.0169%2025.4572%20101.149V101.523Z'%20fill='%2303020D'/%3e%3cpath%20d='M59.2416%200L26.035%2060.8051L19.5433%2056.2846L50.2687%200H59.2416ZM59.7174%200L92.924%2060.8051L99.4158%2056.2846L68.6904%200H59.7174Z'%20fill='url(%23paint0_linear_991_532)'/%3e%3cpath%20d='M92.4822%2061.145L59.4795%2035.3479L26.4769%2061.145L33.2406%2065.8694L59.4795%2045.3405L85.7525%2065.8694L92.4822%2061.145Z'%20fill='url(%23paint1_linear_991_532)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_532'%20x1='30.8116'%20y1='77.039'%20x2='86.368'%20y2='15.3375'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_532'%20x1='41.5883'%20y1='81.0275'%20x2='74.4237'%20y2='44.5601'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
  <a href="/"
     class="flex items-center gap-2 justify-center ui-theme-dark text-center p-2 bg-neutral-1300 rounded order-4"
  >
    <img src="data:image/svg+xml,%3csvg%20width='119'%20height='128'%20viewBox='0%200%20119%20128'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M38.5427%20116.818V73.7886H46.3261V89.3552C48.9432%2086.8401%2052.342%2085.4805%2055.9108%2085.4805C64.3059%2085.4805%2071.7833%2091.8704%2071.7833%20101.523C71.7833%20111.176%2064.3059%20117.566%2055.9108%20117.566C52.1381%20117.566%2048.5693%20116.07%2045.9522%20113.351V116.784H38.5427V116.818ZM64%20101.523C64%2096.1189%2060.1593%2092.3802%2055.163%2092.3802C50.2687%2092.3802%2046.496%2095.9489%2046.3261%20101.149V101.523C46.3261%20106.927%2050.1667%20110.666%2055.163%20110.666C60.1593%20110.666%2064%20106.927%2064%20101.523ZM75.3521%20116.818V73.7886H83.1354V116.818H75.3521ZM94.0117%20128L98.5321%20117.192L86.2283%2086.2283H94.6234L102.475%20107.845L110.428%2086.2283H118.959L102.271%20128H94.0117ZM26.205%2086.2283V90.103C23.5199%2087.146%2019.7472%2085.4805%2015.8725%2085.4805C7.44344%2085.4805%200%2091.8704%200%20101.523C0%20111.176%207.44344%20117.566%2015.8725%20117.566C19.8832%20117.566%2023.7238%20115.798%2026.4769%20112.671V116.818H33.2406V86.2283H26.205ZM25.4572%20101.523C25.4572%20106.859%2021.6166%20110.666%2016.6203%20110.666C11.624%20110.666%207.78332%20106.825%207.78332%20101.523C7.78332%2096.2209%2011.624%2092.3802%2016.6203%2092.3802C21.5146%2092.3802%2025.2873%2096.0169%2025.4572%20101.149V101.523Z'%20fill='white'/%3e%3cpath%20d='M59.2416%200L26.035%2060.8051L19.5433%2056.2846L50.2687%200H59.2416ZM59.7174%200L92.924%2060.8051L99.4158%2056.2846L68.6904%200H59.7174Z'%20fill='url(%23paint0_linear_991_528)'/%3e%3cpath%20d='M92.4822%2061.145L59.4796%2035.3479L26.4769%2061.145L33.2406%2065.8694L59.4796%2045.3405L85.7525%2065.8694L92.4822%2061.145Z'%20fill='url(%23paint1_linear_991_528)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_528'%20x1='30.8116'%20y1='77.039'%20x2='86.368'%20y2='15.3375'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_528'%20x1='41.5883'%20y1='81.0275'%20x2='74.4237'%20y2='44.5601'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
  </a>
</div>
`;

exports[`Components/Logo WithBadge smoke-test 1`] = `
<div class="grid grid-cols-1 sm:grid-cols-2 gap-4">
  <div class="text-center ui-text-code2 order-1">
    light
  </div>
  <div class="text-center ui-text-code2 order-3 sm:order-2">
    dark
  </div>
  <a href="/"
     class="flex items-center gap-2 justify-center text-center p-2 bg-neutral-000 rounded order-2 sm:order-3"
  >
    <img src="data:image/svg+xml,%3csvg%20width='204'%20height='64'%20viewBox='0%200%20204%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M125.844%2049.9572V8.17627H133.387V23.3024C135.942%2020.87%20139.233%2019.5209%20142.708%2019.5209C150.864%2019.5209%20158.1%2025.7349%20158.1%2035.0967C158.1%2044.4586%20150.864%2050.693%20142.708%2050.693C139.049%2050.693%20135.594%2049.2417%20133.019%2046.6049V49.9572H125.844ZM150.557%2035.0967C150.557%2029.8639%20146.837%2026.205%20141.972%2026.205C137.23%2026.205%20133.571%2029.6799%20133.387%2034.7288V35.0967C133.387%2040.3296%20137.107%2043.9885%20141.972%2043.9885C146.837%2043.9885%20150.557%2040.3296%20150.557%2035.0967ZM161.595%2049.9572V8.17627H169.138V49.9572H161.595ZM179.685%2060.7908L184.08%2050.3047L172.142%2020.2568H180.298L187.902%2041.2494L195.629%2020.2568H203.907L187.718%2060.8112H179.685V60.7908ZM113.866%2020.2568V24.0383C111.249%2021.1766%20107.59%2019.5413%20103.829%2019.5413C95.6735%2019.5413%2088.4374%2025.7553%2088.4374%2035.1172C88.4374%2044.4995%2095.6735%2050.693%20103.829%2050.693C107.734%2050.693%20111.433%2048.9965%20114.111%2045.9508V49.9776H120.693V20.2568H113.866ZM113.13%2035.0967C113.13%2040.2683%20109.41%2043.9885%20104.545%2043.9885C99.6799%2043.9885%2095.9596%2040.2683%2095.9596%2035.0967C95.9596%2029.9252%2099.6799%2026.205%20104.545%2026.205C109.287%2026.205%20112.946%2029.7413%20113.13%2034.7288V35.0967Z'%20fill='%2303020D'/%3e%3cpath%20d='M38.5717%200L6.29575%2059.0738L0%2054.6586L29.8639%200H38.5717ZM39.0214%200L71.2973%2059.0738L77.5931%2054.6586L47.7292%200H39.0214Z'%20fill='url(%23paint0_linear_991_548)'/%3e%3cpath%20d='M70.8476%2059.4213L38.7965%2034.3201L6.74542%2059.4213L13.2865%2064L38.7965%2044.0294L64.3066%2064L70.8476%2059.4213Z'%20fill='url(%23paint1_linear_991_548)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_548'%20x1='10.9472'%20y1='74.8439'%20x2='64.9206'%20y2='14.9005'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_548'%20x1='21.4168'%20y1='78.7187'%20x2='53.3166'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
    <div class="inline-flex gap-1 items-center focus-base transition-colors select-none uppercase h-8 bg-transparent dark:bg-transparent rounded border border-neutral-400 dark:border-neutral-900 text-lg p-2 font-semibold text-neutral-800 dark:text-neutral-500">
      <span class="whitespace-nowrap tracking-[0.04em] leading-[20px]">
        docs
      </span>
    </div>
  </a>
  <a href="/"
     class="flex items-center gap-2 justify-center ui-theme-dark text-center p-2 bg-neutral-1300 rounded order-4"
  >
    <img src="data:image/svg+xml,%3csvg%20width='205'%20height='64'%20viewBox='0%200%20205%2064'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M126.751%2049.9572V8.17627H134.294V23.3024C136.849%2020.87%20140.14%2019.5209%20143.615%2019.5209C151.771%2019.5209%20159.007%2025.7349%20159.007%2035.0967C159.007%2044.4586%20151.771%2050.693%20143.615%2050.693C139.956%2050.693%20136.501%2049.2417%20133.926%2046.6049V49.9572H126.751ZM151.464%2035.0967C151.464%2029.8639%20147.744%2026.205%20142.879%2026.205C138.137%2026.205%20134.478%2029.6799%20134.294%2034.7288V35.0967C134.294%2040.3296%20138.014%2043.9885%20142.879%2043.9885C147.744%2043.9885%20151.464%2040.3296%20151.464%2035.0967ZM162.502%2049.9572V8.17627H170.045V49.9572H162.502ZM180.592%2060.7908L184.987%2050.3047L173.05%2020.2568H181.205L188.809%2041.2494L196.536%2020.2568H204.814L188.625%2060.8112H180.592V60.7908ZM114.773%2020.2568V24.0383C112.157%2021.1766%20108.498%2019.5413%20104.737%2019.5413C96.5807%2019.5413%2089.3447%2025.7553%2089.3447%2035.1172C89.3447%2044.4995%2096.5807%2050.693%20104.737%2050.693C108.641%2050.693%20112.341%2048.9965%20115.018%2045.9508V49.9776H121.6V20.2568H114.773ZM114.037%2035.0967C114.037%2040.2683%20110.317%2043.9885%20105.452%2043.9885C100.587%2043.9885%2096.8669%2040.2683%2096.8669%2035.0967C96.8669%2029.9252%20100.587%2026.205%20105.452%2026.205C110.194%2026.205%20113.853%2029.7413%20114.037%2034.7288V35.0967Z'%20fill='white'/%3e%3cpath%20d='M39.4789%200L7.20298%2059.0738L0.907227%2054.6586L30.7712%200H39.4789ZM39.9286%200L72.2046%2059.0738L78.5003%2054.6586L48.6364%200H39.9286Z'%20fill='url(%23paint0_linear_991_544)'/%3e%3cpath%20d='M71.7548%2059.4213L39.7037%2034.3201L7.65265%2059.4213L14.1937%2064L39.7037%2044.0294L65.2138%2064L71.7548%2059.4213Z'%20fill='url(%23paint1_linear_991_544)'/%3e%3cdefs%3e%3clinearGradient%20id='paint0_linear_991_544'%20x1='11.8545'%20y1='74.8439'%20x2='65.8278'%20y2='14.9005'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3clinearGradient%20id='paint1_linear_991_544'%20x1='22.324'%20y1='78.7187'%20x2='54.2238'%20y2='43.2904'%20gradientUnits='userSpaceOnUse'%3e%3cstop%20stop-color='%23FF5416'/%3e%3cstop%20offset='0.2535'%20stop-color='%23FF5115'/%3e%3cstop%20offset='0.461'%20stop-color='%23FF4712'/%3e%3cstop%20offset='0.6523'%20stop-color='%23FF350E'/%3e%3cstop%20offset='0.8327'%20stop-color='%23FF1E08'/%3e%3cstop%20offset='1'%20stop-color='%23FF0000'/%3e%3c/linearGradient%3e%3c/defs%3e%3c/svg%3e"
         width="96px"
         alt="Ably logo"
    >
    <div class="inline-flex gap-1 items-center focus-base transition-colors select-none uppercase h-8 bg-transparent dark:bg-transparent rounded border border-neutral-400 dark:border-neutral-900 text-lg p-2 font-semibold text-neutral-800 dark:text-neutral-500">
      <span class="whitespace-nowrap tracking-[0.04em] leading-[20px]">
        docs
      </span>
    </div>
  </a>
</div>
`;
